### PR TITLE
revised description for Concatenate Arrays with .concat

### LINKED ISSUE #10511: Concatenate with concat, more than one array
#### Challenge Name

https://www.freecodecamp.com/challenges/concatenate-arrays-with-concat
#### Issue Description

The description says that the method can be used to concatenate two arrays. I bet most of us did wonder then how can you concatenate more than two.
I think it is worth mentioning that it can be used to concatenate as many arrays as you want by adding parameters to the method. In fact, the example would gain value by including more arrays. The method is so simple that a more complex example would be an improvement.
#### Browser Information

- Browser Name, Version: 
- Operating System: 
- Mobile, Desktop, or Tablet: 
#### Your Code

``` js



```

### PR DESCRIPTION
**Pull request by johnnyqbui**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #10511

#### Description

Revised description for Concatenate Arrays with .concat challenge for a more accurate description of .concat.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json [hunk 1]
```diff
@@ -563,10 +563,11 @@
       "id": "cf1111c1c16feddfaeb3bdef",
       "title": "Concatenate Arrays with .concat",
       "description": [
-        "<code>concat</code> can be used to merge the contents of two arrays into one.",
+        "<code>concat</code> can be used to merge the contents of two or more arrays into one.",
         "<code>concat</code> takes an array as an argument and returns a new array with the elements of this array concatenated onto the end.",
-        "Here is an example of <code>concat</code> being used to concatenate <code>otherArray</code> onto the end of <code>oldArray</code>:",
-        "<code>newArray = oldArray.concat(otherArray);</code>",
+        "Here are examples of <code>concat</code> being used to concatenate array(s):",
+        "Ex. 1:\n<code>otherArray</code> onto the end of <code>oldArray</code>:\n<code>newArray = oldArray.concat(otherArray);</code>",
```
Comment (Bouncey) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: Do not abbreviate `Example`. Many people using freeCodeCamp currently are not native English speakers

Comment (Bouncey) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: `\n` shouldn't be used. Instead, add another element to the `description` array for new lines.

This helps keep the seed JSON's readable

Comment (Bouncey) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: The code example would be better off inside a `<blockquote>` without `<code>` tags

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json [hunk 2]
```diff
@@ -563,10 +563,11 @@
...
+        "Ex. 2:\n<code>newArray</code> with all values of <code>arr1, arr2,</code> and <code>arr3</code>:\n<code>newArray = arr1.concat(arr2, arr3);</code>",
```
Comment (Bouncey) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: Same as Example 1

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json (hunk 2 above)
Comment (johnnyqbui) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: @Bouncey I've made the requested changes, but was having issues with the branch name so I had to delete my branch name and create a new one. Can I resubmit another PR?

**Pull request closed without merging**